ffmpeg.c

```diff
@@ -578,8 +579,8 @@ static int ffmpeg_read_comments(struct input_plugin_data *ip_data, struct keyval
 	GROWING_KEYVALS(c);
 
 	ffmpeg_read_metadata(&c, ic->metadata);
-	for (unsigned i = 0; i < ic->nb_streams; i++) {
-		ffmpeg_read_metadata(&c, ic->streams[i]->metadata);
+	for (j = 0; j < ic->nb_streams; j++) {
+		ffmpeg_read_metadata(&c, ic->streams[j]->metadata);
```

Comment:
Was supposed to fix the compilation error related to -std, by not mixing code and declarations. Both ugly fixes, but I didn't know how to fix it properly. Thought you'd probably know better. Consider this a documentation of the bug.